content/advanced/330_servicemesh_using_appmesh/integrate_observability/container_insight.md

```diff
@@ -0,0 +1,21 @@
+---
+title: "Container Insights"
+date: 2020-01-27T08:30:11-07:00
+weight: 91
+draft: false
+---
+
+#### Container Insights
+
+CloudWatch Container Insights is a fully managed service that collects, aggregates, and summarizes Amazon EKS metrics and logs. The CloudWatch Container Insights dashboard gives you access to the following information:
```

Comment:
This bulleted list is misformatted (needs a bare newline)

Reply:
updated